### ISSUE TITLE
Question, how to use a patch version of alpine, 3.16.2

### ISSUE THREAD

**Issue by LarsKrogJensen** [chainguard-dev/apko#378]
Using apko with:
`
repositories:
    - https://dl-cdn.alpinelinux.org/alpine/v3.16/main
`
but to have reproducible builds it would be good if I could use `v3.16.2` but that does not seem to exist.

Probably not an apko issue, but hoping someone might know how to achieve that.

Also, I assume v3.16 gets updated when there is a new patch release, or have I misunderstood that

**Comment by panekj** [comment]
Patch releases are not applicable to package repository, they have only meaning for ISO/tarballs distribution.
All packages are patched as soon as possible.

**Comment by panekj** [comment]
https://github.com/chainguard-dev/apko/issues/185

**Comment by sanmai-NL** [comment]
@LarsKrogJensen No, since you can specify the exact versions of packages. When the mirrored packages change version as they do with Alpine Linux, the build breaks (so no silent non-repeatability). If you want such builds to succeed, @panekj has mentioned the open issue for that.

@LarsKrogJensen Can you close this issue now, then?
